Firestore/Example/LibFuzzer.podspec

```diff
@@ -0,0 +1,55 @@
+# Copyright 2017 Google
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# A podspec for libFuzzer. Excludes the 'FuzzerMain.cpp' because the pod
+# installation would require the 'LLVMFuzzerTestOneInput' function to be
+# linked when the pod is being created, but it will be available in
+# the fuzzing application. Hence, users of this Pod are required to
+# provide their main function similar to 'FuzzerMain.cpp'.
+
+Pod::Spec.new do |s|
+  s.name             = 'LibFuzzer'
+  s.version          = '1.0'
+  s.summary          = 'libFuzzer for fuzz testing'
+  s.description      = <<-DESC
+Google's packaging of libFuzzer.
+                       DESC
+  s.homepage         = 'https://llvm.org/docs/LibFuzzer.html'
+  s.license          = { :type => 'BSD' }
+  s.authors          = 'Google, Inc.'
+
+  # Check out only libFuzzer folder.
+  s.source           = {
+    :svn => 'https://llvm.org/svn/llvm-project/compiler-rt/trunk/lib/fuzzer'
+  }
+
+  # Add all source files, except for the FuzzerMain.cpp.
+  s.source_files = '*.{h,cpp,def}'
```

Comment:
Nit: some equals signs are lined up (above), some aren't, I'd suggest either lining them up consistently (within file, or within paragraph) or not at all.

Reply:
Aligned all equal signs.